### PR TITLE
Added LWIMIQ entry for March 24 2017

### PR DESCRIPTION
**Pull request by syncrou**

### CODE REVIEW

**Comment by syncrou** [comment]
@JPrause - Here is the latest LWMIQ post. 

@d-m-u, @imtayadeway and/or others please review if you could.

**Comment by JPrause** [comment]
@syncrou sweet,...thanks for your submission.

**Comment by JPrause** [comment]
Wow @syncrou! You are giving @d-m-u a run for the money.
"PR should be taken together with alcohol",...who knew?

LGTM! One for the ages!
@gtanzillo and to think,..you never told us that @syncrou was a doctor?

**Comment by d-m-u** [comment]
👍 Amazing.

File: site/_posts/2017-03-24-pontificatory-requestiveous.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+introduced me to PR.  My life is changed! Those Tower objects are now created at startup!
+
+## Some have Noticed Their Zeds Removed
+
+### [Split metric collections into smaller intervals](https://github.com/ManageIQ/manageiq/pull/14332)
+[Greg B](https://github.com/blomquisg): Large metric collections.  That's all I remember before I started on PR.
+The intervals between `last_perf_capture_on` and `today` were too lengthy.
+We all get up every day and make decisions. One day I got up and decided to take PR.
+Those smaller metric collections are thanking me, but the real truth. The real truth is that I'm
+thanking myself! Thanks PR! Because of you my Bzeds are removed.
```
Comment (imtayadeway) [inline site/_posts/2017-03-24-pontificatory-requestiveous.md]: :heart: :bee: :zzz:

**Comment by imtayadeway** [comment]
:+1: Lovely!

File: site/_posts/2017-03-24-pontificatory-requestiveous.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+title: "Last Week in ManageIQ: Pontificatory Requestiveous - PR"
+author: syncrou
+date: 2017-03-24 13:34:37 -04:00
```
Comment (imtayadeway) [inline site/_posts/2017-03-24-pontificatory-requestiveous.md]: Not sure, but do we want to update the date to reflect published date (presumably) of tomorrow? Wondering if I could update the generator to make this `today.tuesday? ? today : next_tuesday` :thinking:

**Comment by JPrause** [comment]
@syncrou did you have any further edits based on Tim's comments?

**Comment by syncrou** [comment]
@JPrause - Yes - I should get the date updated. I'll get that out.

**Comment by syncrou** [comment]
@JPrause - All edits should be in.

File: site/_posts/2017-03-28-pontificatory-requestiveous.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Hi, [Dr. Drew Syncrounious](https://github.com/syncrou) here to talk with you about an important new solution just released into 
+[master](https://github.com/manageiq/manageiq). Known scientifically as Pontificatory Requestiveous, for us in the industry, it is affectionately called:
+
+[PR](https://help.github.com/articles/about-pull-requests/).
+
+A few questions
+
+Do you struggle with [memory loss](http://stackoverflow.com/questions/23281331/how-to-clean-up-allocated-memory-in-c)?
+
+Overly terse [varible names](https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js)?
```
Comment (d-m-u) [inline site/_posts/2017-03-28-pontificatory-requestiveous.md]: Variable?

Comment (JPrause) [inline site/_posts/2017-03-28-pontificatory-requestiveous.md]: @syncrou did you need to modify something based on Drew's comment before we merge?

Comment (syncrou) [inline site/_posts/2017-03-28-pontificatory-requestiveous.md]: Yes - I'll get that in now.

**Comment by syncrou** [comment]
@JPrause - All the edits should be in.

**Comment by JPrause** [comment]
@cybette we're ready for the merge.

**Comment by cybette** [comment]
Loved reading this!

**Pull request merged**